md/utils.py

```diff
@@ -0,0 +1,32 @@
+from collections import OrderedDict
+
+
+class GroupedData(object):
```

Comment:
That was a casualty of the merge. Should have already been moved to tsdata. Will move it again.